.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 wp-config.php
 .htaccess
+wp-content/uploads
```

Comment:
I think that we should leave this out for 2 reasons.
1) It is valuable to learn how to add an item to your .gitignore file (and have a conversation around what that is) 
2) If WPEngine isn't used and the uploads directory is light - they may not need to remove it at all.